demo/scripts/script-test.js

```diff
@@ -1,4 +1,5 @@
 var notyf;
+var notyfNotifications = [];
```

Comment:
Currently maintaining the active notification here so that they can be accessed easily to test dismiss functionality for them. Not the most elegant solution, so I'm not very sure of this.
Another way could be listing all the active notifications separately, and showing a dismiss button beside them, but clearly, that would require quite a lot of work.
What are your thoughts about this? What could be done instead? @caroso1222

Reply:
I see. That's a creative solution. 

Now, I would like to keep the tests far from touching the global directly. I like that you understand how the `script-test.js` is related to the cypress `tests`, I also like the idea of keeping a list of notifications.

My suggestion would be to follow the same principle for `dismissAll`. Create a method `dismiss(idx)` which would dismiss the `idx` notification from the `notifications` array. And also create a button and text input for this, so you can interact with it from the UI (and from cypress).

Something like this:

```js
document.getElementById('dismiss-btn')
        .addEventListener('click', function(){
          const idx = document.getElementById('notification-idx').value;
          dismiss(idx);
        });

function dismiss(idx) {
  notyf.dismiss(notyfNotifications[idx]);
  notyfNotifications.splice(idx, 1);
}
```